README.md

```diff
@@ -9,13 +9,19 @@ The DURI ([GA4GH::DURI](http://ga4gh-duri.github.io)) Work Stream is one of the
 
 The GA4GH DURI workstream is led by [Jaime Guidry Auvil](https://datascience.cancer.gov/content/jaime-m-guidry-auvil-phd) and [Tommi Nyronen](https://www.linkedin.com/in/nyronen/). All members of this team must follow the [GA4GH Standards of Professional Conduct](https://www.ga4gh.org/wp-content/uploads/GA4GH-Standards-for-Professional-Conduct_22-Jan-2018.pdf).
 
+This team has two main standards:
+* The [Data Use Ontology (DUO)](https://github.com/EBISPOT/DUO) for semantic tagging of datasets with usesage restrictions
+* [GA4GH Passports](https://github.com/ga4gh-duri/ga4gh-duri.github.io/tree/master/researcher_ids) which is a digital identity for researchers to access federated data with standard definitions and ways to express data access rights to support consistency.
```

Comment:
Perhaps just simply to:

"which is a standards-based digital identity for researchers to access distributed data that includes ways to express data access rights"

Reply:
Updated.